workflow-templates/release.yml

```diff
@@ -1,4 +1,4 @@
-name: github-release
```

Comment:
question: is there a reason that it's no longer `github-release` and now just `release`?

Reply:
this will now do an npm release now in addition to a GitHub Release.